crates/core/tedge_mapper/src/c8y/mapper.rs

```diff
@@ -87,6 +105,12 @@ impl TEdgeComponent for CumulocityMapper {
             tc.forward_from_local("q/us/#", local_prefix.clone(), "")?;
             tc.forward_from_local("c/us/#", local_prefix.clone(), "")?;
 
+            // SmartREST1
+            tc.forward_from_local("s/ul/#", local_prefix.clone(), "")?;
+            tc.forward_from_local("t/ul/#", local_prefix.clone(), "")?;
+            tc.forward_from_local("q/ul/#", local_prefix.clone(), "")?;
+            tc.forward_from_local("c/ul/#", local_prefix.clone(), "")?;
```

Comment:
Why not guard this block with an `if !use_certificate`, as there's no point adding these rules when using certificates.

Reply:
quickly done in [452943b](https://github.com/thin-edge/thin-edge.io/pull/3196/commits/452943bf7b9d8933c5d20513d813259e2389116d)